commands/station.js

```diff
@@ -64,7 +65,7 @@ export const station = async ({ json, experimental }) => {
         })
       },
       onMetrics: m => metrics.submit('zinnia', m)
-    })
+    }), { retries: 1000 })
```

Comment:
I tried `Infinity`, unfortunately that doesn't work. See https://github.com/sindresorhus/p-retry/issues/58

Reply:
Can we use something like a max-safe integer?

Reply:
it also fails with that unfortunately